### DISCUSSION TITLE
在Modal、FormDialog中使用Select，Options有问题

### DISCUSSION THREAD

**Discussion by Chen-Weidong** [alibaba/formily#2499] [Q&A 2.x] [answered]
在Modal、FormDialog中使用Select的时候，如果移动滚动条，Select组件中的Options不会跟着组件移动。
可以看这个sandbox  [https://codesandbox.io/s/practical-mayer-o7bv4?file=/App.tsx](url)
![image](https://user-images.githubusercontent.com/73837770/144960618-6e28f4b5-dba1-4b34-8140-5c3cf77a818a.png)

**Comment by janryWang** [comment, marked as answer]
这个跟formily没关系，是antd的使用问题，我记得是有个API专门指定浮层渲染容器的，具体看antd文档

**Reply by Chen-Weidong** [reply]
好的，谢谢
